reference/configuration/framework.rst

```diff
@@ -526,10 +526,19 @@ enabled
 
 Whether or not to enable the ``translator`` service in the service container.
 
-fallback
-........
+.. _fallback:
 
-**type**: ``string`` **default**: ``en``
+fallbacks
+.........
+
+.. versionadded:: 2.3.25
+    The ``fallbacks`` option was introduced in Symfony 2.3.25. Prior
+    to Symfony 2.3.25, it was called ``fallback`` and only allowed one fallback
+    language defined as a string.
+    Please note that you can still use the old ``fallback`` option if you want
+    define only one fallback.
```

Comment:
The "**type**: ``string|array`` **default**: ``array('en')``" should just be before the `versionadded` directive.